helm-openshift-install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -eq 0 ]; then
+    echo "Usage: $0 <arguments>"
+    exit 1
+fi
+
+START_TIME=$(date +%s)
+TIMEOUT_SEC=25 
+SPINUP_SEC=60 # Helm Upgrade can have unexpected results if it is run while the cluster is initially starting
+
+# Run the install without the generated Openshift Route being passed to Registry Viewer fqdn as it does not exist yet
+helm install devfile-registry deploy/chart/devfile-registry --set global.isOpenShift=true "$@"
```

Comment:
Should `devfile-registry` be customizable here?

Reply:
I can definitely edit it to allow the `devfile-registry` or name in this case be set via an env variable or something, same with the path if you think allowing that to change is also necessary. I suggest an environment variable instead of an argument to the script so we can continue using the simple passing in of the helm install arguments via `"$@". If we allow the name and path to be set via arguments I believe that would be fine we'd just need to parse the arguments first to determine where each needs to go

Reply:
Perhaps we can track it separately? I think it's probably fine to leave it.